libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1555,6 +1558,15 @@ bool RC_Channel::do_aux_function(const AUX_FUNC ch_option, const AuxSwitchPos ch
         mount->set_yaw_lock(ch_flag == AuxSwitchPos::HIGH);
         break;
     }
+
+    case AUX_FUNC::MOUNT_LRF_ENABLE: {
```

Comment:
think about how we can make it happen at startup?

Reply:
I've created an issue to capture this enhancement request https://github.com/ArduPilot/ardupilot/issues/26585

I will go ahead and merge this without this enhancement request included in the code because it is not completely straight forward and I don't want to hold up this enhancement which a Partner is waiting on.